quarterly_report.ipynb

```diff
@@ -10,20 +10,21 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.            lambda x: x.mean() # Changed to accomodate partial data from content gap metrics
</pre><p>I think this is a reasonable approach, but there should be some notification to the notebook user if we are using partial data. Ideally, it would be integrated into the report table, but I don't know a good way to do that, so a printed notice (e.g. using <code>wmfdata.utils.print_err</code>) is probably the most feasible approach.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/movement-metrics/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Since we're not integrating the tables at this point, we can print the notice along with the content gap metrics table.</p>